data/challenges/xr591.challenge

```diff
@@ -1,12 +1,17 @@
 <?xml version="1.0"?>
-<challenge version="2">
+<challenge version="3">
+    <unlock_list list="false"/>
```

Comment:
IMO it shouldn't be necessary to specify that everywhere, it should defaut to normal if absent from the file